src/Tizen.NUI/src/public/Images/ImageLoading.cs

```diff
@@ -137,6 +138,217 @@ public static PixelBuffer LoadImageFromFile(string url)
             return ret;
         }
 
+        /// <summary>
+        /// Load an image synchronously from Stream. Load from stream currnet postion to end of stream.
+        /// </summary>
+        /// <param name="stream">The stream of the image file to load.</param>
+        /// <param name="size">The width and height to fit the loaded image to, 0.0 means whole image.</param>
+        /// <param name="fittingMode">The method used to fit the shape of the image before loading to the shape defined by the size parameter.</param>
+        /// <param name="samplingMode">The filtering method used when sampling pixels from the input image while fitting it to desired size.</param>
+        /// <param name="orientationCorrection">Reorient the image to respect any orientation metadata in its header.</param>
+        /// <returns>Handle to the loaded PixelBuffer object or an empty handle in case loading failed.</returns>
+        /// <exception cref="ArgumentNullException"> Thrown when size is null. </exception>
+        /// <exception cref="InvalidOperationException"> Thrown when stream don't have any data. </exception>
+        /// <remarks>Hidden API: Only for inhouse or developing usage. The behavior and interface can be changed anytime.</remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static PixelBuffer LoadImageFromMemory(Stream stream, Size2D size, FittingModeType fittingMode, SamplingModeType samplingMode, bool orientationCorrection)
```

Comment:
I think you can make the following 4 methods as one method by using default parameter. (reduce duplicated codes)
for example, `public static PixelBuffer LoadImageFromMemory(Stream stream, Size2D size, FittingModeType fittingMode = FittingModeType.Center, SamplingModeType samplingMode = SamplingModeType.NoFilter, bool orientationCorrection = true)`
could you check? ^^;

Reply:
Oh, I love that way!
I will fix `LoadImageFromFile` as same way.